src/test/resources/log4j.xml

```diff
@@ -8,7 +8,8 @@
     <appender name="console" class="org.apache.log4j.ConsoleAppender">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=xml%3AIndentCheck)